tests/phpunit/elementor/modules/atomic-widgets/test-atomic-widget-base.php

```diff
@@ -23,72 +25,153 @@ class Test_Atomic_Widget_Base extends Elementor_Test_Base {
 	/**
 	 * @dataProvider get_atomic_settings_data_provider
 	 */
-	public function test_get_atomic_settings( $prop_types, $settings, $result ) {
+	public function test_get_atomic_settings( $args, $arrange_cb = null ) {
 		// Arrange.
+		$cleanup = null;
+
+		if ( $arrange_cb ) {
+			$cleanup = $arrange_cb();
+		}
```

Comment:
```suggestion
		$cleanup = $arrange_cb ? $arrange_cb() : fn() => {};
```

maybe?

(and then remove the condition at the end)
(but not critical)

Reply:
Beautiful 🌷 Done.